colors/gruvbox_crisp.lua

```
@@ -0,0 +1,11 @@
+-- Safely attempt to load the theme to avoid hard failures when the plugin
+-- is missing from the runtimepath (e.g., when :colorscheme is called early).
+local ok, mod = pcall(require, "gruvbox_crisp")
+if not ok then
+  if vim and vim.notify then
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'vim'

(W113)